### ISSUE TITLE
Error in connectome?

### ISSUE THREAD

**Issue by mattbaker** [Connectome/GoPiGo#5]
Should this be `+=1` in [connectome.py](https://github.com/Connectome/GoPiGo/blob/master/disembodiedConnectome.py#L55)?

```python
def ADAL():
        postSynaptic['ADAR'][nextState] += 2
```

Following @Interintel's work, [SimpleConnectomeUDPv4.py](https://github.com/Interintel/ConnectomeEngine/blob/master/SimpleConnectomeUDPv4.py) uses +1, but [GoPiGoConnectomev5.py](https://github.com/Interintel/ConnectomeEngine/blob/master/GoPiGoConnectomev5.py) uses +2. 

I wonder if it's a typo in the GoPiGo version (it appears that line is [duplicated](https://github.com/Interintel/ConnectomeEngine/issues/1)) in which case `SimpleConnectomeUDPv4.py` may be the source of truth.

Anyone know?

**Comment by ggaabe** [comment]
Hi Matt, we combined the two lines (there were several of these instances) to lighten the computational burden of the accumulated weights from the signaling between neurons. Because the state of all neurons are broken down into time-states between the completions of firings, the result is the same. If we could have a multithreaded function that simultaneously scans all neurons as they are being incremented and fires based on their states represented in each millisecond, we'd also have much more precise behavior.

Before that though, I'd really like to multithread the motor function, as I have described here:
http://stackoverflow.com/questions/32027233/how-to-multithread-multiprocess-just-one-instance-of-a-particular-function-in-py
So yes, you are right. What you have described is, in fact, closer to the truth. Each increment operation represented the firing of a certain number of synapses. But those separations were in place because of the several to hundred milliseconds difference each connection's signal would have in actually reaching the destination neuron.

If we can properly multithread, which we should only be doing on an RPi2, then we could more accurately represent the minute differences in increments. But until then, with time-states that are defined in such large chunks to include the completion of all neuron firings, the difference is non-existent and it is best to speed up those operations as much as possible.
